docs/user/manifest.json

```
@@ -11,6 +11,12 @@
 		"markdown_source": "../user/faq.md",
 		"parent": null
 	},
+	{
+		"title": "Site Health Screen",
+		"slug": "site-health",
+		"markdown_source": "../user/site-health.md",
```

```suggestion
		"markdown_source": "../user/administration/site-health.md",
```

I believe the page would be better positioned inside the ./administration folder and the corresponding link should be added to the ./administration/README.md page.